packages/playground/src/components/deployment_data_dialog.vue

```diff
@@ -71,7 +71,7 @@
               <CopyReadonlyInput
                 v-for="disk of contract.mounts"
                 :key="disk.name"
-                :label="getDiskLabel(contract, disk)"
+                :label="getDiskLabel(disk)"
```

Comment:
```suggestion
                :label="`Disk ${disk.name}`"
```
no need to have ea function for this